examples/grpc/build.gradle

```diff
@@ -16,6 +16,7 @@ dependencies {
     testImplementation libs.awaitility
     testImplementation libs.junit5.jupiter.api
     testImplementation libs.protobuf.java.util
+    testImplementation libs.testcontainers.junit.jupiter
```

Comment:
Please create a separate project for the example you're writing. Don't make an example do more than one thing.

Reply:
Well, am I right to understand?
Are you telling me to create and use a grpc-reverse-proxy project in the sample directory?